server/views/components/hearing-notes/template.njk

```diff
@@ -1,17 +1,26 @@
 <div class="app-summary-card__body">
 
     <div class="govuk-grid-row govuk-grid-column-full">
+        <table class="govuk-table" id="previous-note">
```

Comment:
remove the id field on the table. this is causing duplicate id fields resulting in a11y checks failures hence the integration tests are failing